RepositoryTests/TSQLRepositoryTests.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using NUnit.Framework;
+using Moq;
+using UBB_SE_2024_Team_42.Repository.TSQLRepository;
+using UBB_SE_2024_Team_42.Domain.Notification;
+using UBB_SE_2024_Team_42.Domain.Category;
+using UBB_SE_2024_Team_42.Domain.Badge;
+using UBB_SE_2024_Team_42.Domain.User;
+using UBB_SE_2024_Team_42.Domain.Post;
+using UBB_SE_2024_Team_42.Domain.Tag;
+using UBB_SE_2024_Team_42.Domain.Question;
+using UBB_SE_2024_Team_42.Domain.Reaction;
+using System.Collections;
+using System.Reflection;
+
+namespace Team42Test.RepositoryTests
+{
+    [TestFixture]
+    public class TSQLRepositoryTests
+    {
+        public TSQLRepository mockTsqlRepository;
+        [SetUp]
+        public void Setup()
+        {
+            mockTsqlRepository = new TSQLRepository();
+        }
+        [Test]
+        public void GetNotificationsOfUser_ReturnsNotifications()
```

Comment:
Numele de teste ar trebui sa fie schimbate sa fie puțin mai explicite
Un exemplu pt asta ar fi
GetNotificationOfUser_ValidUserIDProvided_ReturnsIEnumerableOfINotification